### PR TITLE
PMM-6824 DBaaS restart

### PR DESCRIPTION
**Pull request by askomorokhov**
[PMM-6824](https://jira.percona.com/browse/PMM-6824)

- [x] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1234

- https://github.com/percona-platform/dbaas-api/pull/63
- https://github.com/Percona-Lab/pmm-api-tests/pull/143
- https://github.com/percona-platform/dbaas-controller/pull/90
- https://github.com/percona/pmm-managed/pull/517
- https://github.com/percona/pmm/pull/611
---
- [x] Tests passed.
- [ ] Feature build pass.
- [x] (Re)requested review.
- [x] Fix conflicts with target branch.
- [ ] Update API dependency.
- [x] Update jira ticket description if needed.
- [x] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

**Review by percona-csalguero — APPROVED** [review]

**Review by JiriCtvrtka — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

File: services/dbaas/dbaas_client.go [hunk 1]
```diff
@@ -68,6 +68,11 @@ func (c *Client) DeleteXtraDBCluster(ctx context.Context, in *controllerv1beta1.
 	return c.xtradbClusterClient.DeleteXtraDBCluster(ctx, in, opts...)
 }
 
+// RestartXtraDBCluster restarts XtraDB cluster.
+func (c *Client) RestartXtraDBCluster(ctx context.Context, in *controllerv1beta1.RestartXtraDBClusterRequest, opts ...grpc.CallOption) (*controllerv1beta1.RestartXtraDBClusterResponse, error) {
```
Comment (AlekSi) [inline services/dbaas/dbaas_client.go:72]: @askomorokhov @BupycHuk It is probably too late for this PR, but why we pass `opts ...grpc.CallOption`?

**Review by AlekSi — APPROVED** [review]

File: services/dbaas/dbaas_client.go (hunk 1 above)
Comment (BupycHuk) [reply services/dbaas/dbaas_client.go:72]: Just to keep ability to pass them. it's not required at the moment.

Comment (AlekSi) [reply services/dbaas/dbaas_client.go:72]: Then I don't quite understand why we need that wrapper that just passes all parameters as-is, without any logic, adoption, etc.

Let's drop opts once we have a chance, and then maybe revisit if we need that wrapper at all

Comment (askomorokhov) [reply services/dbaas/dbaas_client.go:72]: OK, I'll try to remove opts here.

Comment (AlekSi) [reply services/dbaas/dbaas_client.go:72]: Only if it can be done quickly in that PR

Comment (askomorokhov) [reply services/dbaas/dbaas_client.go:72]: It requires many changes in many places. Better to do this without hurrying.

**Comment by AlekSi** [comment]
@askomorokhov more conflicts

**Pull request merged**